modules/mobile-agents/windows/jax-rs/src/main/java/org/wso2/carbon/mdm/mobileservices/windows/common/Constants.java

```diff
@@ -24,140 +24,174 @@
-		public static final String SYNCML_SOURCE_URI = "SOURCE_URI";
-		public static final String SYNCML_TARGET_URI = "TARGET_URI";
-		public static final String SYNCML_MESSAGE_ONE = "1";
-		public static final String SYNCML_MESSAGE_TWO = "2";
-	}
+    //Service endpoints
+    public static final String DISCOVERY_SERVICE_ENDPOINT =
+            "org.wso2.carbon.mdm.mobileservices.windows.services.discovery.DiscoveryService";
+    public static final String CERTIFICATE_ENROLLMENT_SERVICE_ENDPOINT =
+            "org.wso2.carbon.mdm.mobileservices.windows.services.wstep" +
+            ".CertificateEnrollmentService";
+    public static final String CERTIFICATE_ENROLLMENT_POLICY_SERVICE_ENDPOINT =
+            "org.wso2.carbon.mdm.mobileservices.windows.services.xcep" +
+            ".CertificateEnrollmentPolicyService";
+
+    //Services' target namespaces
+    public static final String DISCOVERY_SERVICE_TARGET_NAMESPACE =
+            "http://schemas.microsoft.com/windows/management/2012/01/enrollment";
+    public static final String DEVICE_ENROLLMENT_SERVICE_TARGET_NAMESPACE =
+            "http://schemas.microsoft.com/windows/pki/2009/01/enrollment/RSTRC";
+    public static final String CERTIFICATE_ENROLLMENT_POLICY_SERVICE_TARGET_NAMESPACE =
+            "http://schemas.microsoft.com/windows/pki/2009/01/enrollmentpolicy";
+
+    //Certificate enrollment service urls and namespaces
+    public static final String WS_TRUST_TARGET_NAMESPACE =
+            "http://docs.oasis-open.org/ws-sx/ws-trust/200512";
+    public static final String WS_SECURITY_TARGET_NAMESPACE =
+            "http://docs.oasis-open.org/wss/2004/01/oasis-200401-wss-wssecurity-secext-1.0.xsd";
+    public static final String SOAP_AUTHORIZATION_TARGET_NAMESPACE =
+            "http://schemas.xmlsoap.org/ws/2006/12/authorization";
+
+    //Certificate enrollment policy service urls and namespaces
+    public static final String ENROLLMENT_POLICY_TARGET_NAMESPACE =
+            "http://schemas.microsoft.com/windows/pki/2009/01/enrollmentpolicy";
+    public static final String REQUEST_WRAPPER_CLASS_NAME =
+            "com.microsoft.schemas.windows.pki._2009._01.enrollmentpolicy.GetPolicies";
+    public static final String RESPONSE_WRAPPER_CLASS_NAME =
+            "com.microsoft.schemas.windows.pki._2009._01.enrollmentpolicy.GetPoliciesResponse";
+
+    //Servlet Context attributes names
+    public static final String CONTEXT_WAP_PROVISIONING_FILE = "WAP_PROVISIONING_FILE";
+    public static final String WINDOWS_PLUGIN_PROPERTIES = "WINDOWS_PLUGIN_PROPERTIES";
+
+    //Message handler constants
+    public static final String CONTENT_LENGTH = "Content-Length";
+    public static final String SECURITY = "Security";
+
+    //Web services media types
+    public static final String SYNCML_MEDIA_TYPE = "application/vnd.syncml.dm+xml;charset=utf-8";
+
+    /**
+     * Discovery service related other constants
+     */
+    public final class Discovery {
```

Comment:
Might also be good if we prevent instantiating this particular class so that it'd be clear to someone how exactly this needs to be used.